grafana-plugin/src/containers/ServiceNowConfigDrawer/ServiceNowConfigDrawer.tsx

```diff
@@ -0,0 +1,442 @@
+              )}
+            />
+
+            <Controller
+              name={'additional_settings.password'}
+              control={control}
+              rules={{ required: 'Password is required' }}
+              render={({ field }) => (
+                <Field
+                  key={'AuthPassword'}
+                  label={'Password'}
+                  invalid={!!errors.additional_settings?.password}
+                  error={errors.additional_settings?.password?.message}
+                >
+                  <Input {...field} type="password" />
+                </Field>
+              )}
+            />
+
+            <HorizontalGroup>
+              <Button className={''} variant="secondary" onClick={onAuthTest}>
+                Test
+              </Button>
+              <div>
+                <RenderConditionally shouldRender={isAuthTestRunning}>
+                  <LoadingPlaceholder text="Loading" className={styles.loader} />
+                </RenderConditionally>
+
+                <RenderConditionally shouldRender={!isAuthTestRunning && authTestResult !== undefined}>
+                  <HorizontalGroup align="center" justify="center">
+                    <Text type="primary">{authTestResult ? 'Connection OK' : 'Connection failed'}</Text>
+                    <Icon name={authTestResult ? 'check-circle' : 'x'} />
+                  </HorizontalGroup>
+                </RenderConditionally>
+              </div>
+            </HorizontalGroup>
+          </div>
+
+          <div className={styles.border}>
+            <VerticalGroup spacing="md">
+              <HorizontalGroup spacing="xs" align="center">
+                <Text type="primary" size="small">
+                  Status Mapping
+                </Text>
+                <Icon name="info-circle" />
+              </HorizontalGroup>
+
+              <table className={'filter-table'}>
+                <thead>
+                  <tr>
+                    <th>OnCall Alert group status</th>
+                    <th>ServiceNow incident status</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  <tr>
+                    <td>Firing</td>
+
+                    <td>
+                      <Controller
+                        name={'additional_settings.state_mapping.firing'}
+                        control={control}
+                        render={({ field }) => (
+                          <Select
+                            {...field}
+                            key="state_mapping.firing"
+                            menuShouldPortal
+                            className="select control"
+                            options={getAvailableStatusOptions(OnCallAGStatus.Firing)}
+                            onChange={(option: SelectableValue) => {
+                              onStatusSelectChange(option, OnCallAGStatus.Firing);
+                              setValue('additional_settings.state_mapping.firing', null);
```

Comment:
temporarily set the value to null, we don't have yet values from backend and not totally sure on the JSON response